src/patients/notes/NoteTab.tsx

```diff
@@ -0,0 +1,96 @@
+/* eslint-disable react/no-danger */
+import React, { useState } from 'react'
+import { useSelector, useDispatch } from 'react-redux'
+import { useTranslation } from 'react-i18next'
+import { Button, List, ListItem, Toast, Alert } from '@hospitalrun/components'
+import NewNoteModal from 'patients/notes/NewNoteModal'
+import Note from 'model/Note'
+import Patient from 'model/Patient'
+import { updatePatient } from 'patients/patient-slice'
+import { RootState } from '../../store'
+import Permissions from '../../model/Permissions'
+
+interface Props {
+  patient: Patient
+}
+
+const NoteTab = (props: Props) => {
```

Comment:
Need to add tests for the notes tab: 

1. renders an add new button for someone with permissions
2. doesn't render the button if no permissions
3. renders a warning message if no notes
4. renders a list of notes
5. Clicking button displays modal
6. Saving calls the update patient action
7. On successful save, a toast appears